enrollment-server/src/main/java/com/wultra/app/enrollmentserver/mock/MockCustomActivationProvider.java

```diff
@@ -17,13 +17,13 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.app.onboardingserver.mock;
+package com.wultra.app.enrollmentserver.mock;
```

Comment:
This is the main change of this PR. Other changes are just merge from the upstream.